recipe/meta.yaml

```diff
@@ -10,15 +10,15 @@ source:
 
 build:
   noarch: python
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I think there was a reason to use that but I cannot recall. Anyway, let's give it a go and maybe the issue no longer exists. If it breaks anything we can pull the package.

Reply:
Looks like it makes a circular dependency, as `pip` requires `setuptools` requires `wheel`.

What we could do is do is:

```bash
python -c "import flit_core.buildapi; flit_core.build_api.build_wheel("tmp")"
python -m installer tmp/*.whl
```

(Though it looks like installer isn't present in conda-forge?)

Reply:
It's called [`python-installer`]( https://github.com/conda-forge/python-installer-feedstock )

Reply:
Those have been there since the feedstock was added. IIUC (and I may be out-of-date) is those are required for `pip` to do source builds or install binary wheels. It's possible things have changed. Though would approach this carefully as changes to the `pip` package would affect a lot of conda-forge packages

Reply:
Those are only required to build wheels, and only if the build backend is setuptools. If that's the case, then setuptools and wheel should be listed in the build requirements of a recipe, just like any other build backend. I wonder if it would be possible to do static analysis to see what recipes are missing a build backend specification? Ideally, setuptools and wheel should not be installed if they are not required.

Getting a strange failure now about a non-existent file being unlinked. 😜

Reply:
Yeah it may be we want to change things. If so, we would likely want to add a (mini?) migrator to update recipes dependencies before making changes to `pip` to minimize disruption